src/store/modules/ADempiere/dictionary/window/actions.js

```diff
@@ -219,25 +219,8 @@ export default {
               })
             },
             loadData: ({ parentUuid: tabAssociatedUuid, containerUuid }) => {
-              const reportDefinition = rootGetters.getStoredReport(process.uuid)
               const storedTab = rootGetters.getStoredTab(windowUuid, tabAssociatedUuid)
               const { table_name } = storedTab
-              if (!isEmptyValue(reportDefinition)) {
-                // auto run report if without parameters
-                if (!reportDefinition.hasParameters || isEmptyValue(reportDefinition.fieldsList)) {
```

Comment:
This code was made to auto generate the reports without opening the modal, when they have no parameters, you have to check why no parameters are arriving for a process/report that in first execution if it showed parameters, also consider the auto execution does not apply to processes.